packages/dd-trace/src/opentracing/tracer.js

```diff
@@ -52,12 +54,32 @@ class DatadogTracer extends Tracer {
     }
   }
 
+  startSpan (name, fields) {
+    if (fields && fields.references) {
```

Comment:
Condition could be simplified to not repeat the fields check and relying more on returning early before any other conditions.

Reply:
I simplified this a bit.